### PR TITLE
ORCA: Change King County Water Taxi to Handheld

### PR DESCRIPTION
**Pull request by cookieguru**
I have verified that this FTP type was used by the handheld readers that the water taxi staff uses.  However, when I tag my card with one of the loaders on 3rd Ave it is showing as Water Taxi coach 3.  @supersat saw the same with the loaders in the tunnel.  All of these handhelds are owned by King County, so it makes sense that they are under the KCM agency ID, but it appears it's not exclusively for the water taxi.

Leaving as a draft now while I make an icon.

### CODE REVIEW

**Comment by codebutler** [comment]
@cookieguru hey, I'd love to get this in. 

For an icon, something like this?
https://thenounproject.com/mikicon/collection/contactless-payment/?i=636921
https://material.io/resources/icons/?search=phone&icon=phonelink_ring&style=baseline

**Comment by codebutler** [comment]
@cookieguru hey, any update?

**Comment by cookieguru** [comment]
The first one is similar to what I was envisioning.  I think the only hurdle at this point is converting it to Andriod's drawable format; can you help with that?

**Comment by codebutler** [comment]
Here you go!

I paid for the file, so no need to worry about attribution.


```xml
<vector xmlns:android="http://schemas.android.com/apk/res/android"
    android:width="100dp"
    android:height="100dp"
    android:viewportWidth="100"
    android:viewportHeight="100">
  <path
      android:pathData="M69.2,43.4c0,-3.5 -2.9,-6.4 -6.4,-6.4H37.2c-3.5,0 -6.4,2.9 -6.4,6.4v36.2c0,3.5 2.9,6.4 6.4,6.4h25.7c3.5,0 6.4,-2.9 6.4,-6.4V43.4zM62.8,76.7c0,0.7 -0.5,1.2 -1.2,1.2H38.4c-0.7,0 -1.2,-0.5 -1.2,-1.2V44.9c0,-0.7 0.5,-1.2 1.2,-1.2h23.2c0.7,0 1.2,0.5 1.2,1.2V76.7z"
      android:fillColor="#000000"/>
  <path
      android:pathData="M65.9,22.8c0.3,-0.3 0.4,-0.6 0.4,-1c0,-0.3 -0.1,-0.7 -0.4,-1c-4.2,-4.3 -9.9,-6.7 -15.8,-6.8s-11.7,2.3 -16,6.5c-0.6,0.6 -0.6,1.4 0,2c0.6,0.6 1.4,0.6 2,0c3.8,-3.7 8.7,-5.7 14,-5.7c5.3,0.1 10.2,2.1 13.9,5.9C64.5,23.3 65.3,23.3 65.9,22.8z"
      android:fillColor="#000000"/>
  <path
      android:pathData="M60.9,27.7c0.3,-0.3 0.4,-0.6 0.4,-1s-0.1,-0.7 -0.4,-1c-2.9,-3 -6.8,-4.6 -10.9,-4.7c-4,0 -7.9,1.6 -10.9,4.5c-0.6,0.6 -0.6,1.4 0,2s1.4,0.6 2,0c2.4,-2.4 5.6,-3.7 9,-3.6c3.4,0 6.5,1.4 8.9,3.8C59.5,28.3 60.3,28.3 60.9,27.7z"
      android:fillColor="#000000"/>
  <path
      android:pathData="M44,30.6c-0.5,0.5 -0.5,1.4 0.1,2s1.4,0.6 2,0c2.2,-2.2 5.8,-2.1 8,0.1c0.6,0.6 1.4,0.6 2,0c0.3,-0.3 0.4,-0.6 0.4,-1c0,-0.3 -0.1,-0.7 -0.4,-1C52.7,27.4 47.3,27.3 44,30.6z"
      android:fillColor="#000000"/>
</vector>
```

**Comment by cookieguru** [comment]
Thanks, that helps immensely.  Ready for review 🙂

File: farebot-transit-orca/src/main/java/com/codebutler/farebot/transit/orca/OrcaTrip.java [hunk 1]
```diff
@@ -635,6 +635,9 @@ public String getStartStationName(@NonNull Resources resources) {
         } else if (getFTPType() == OrcaTransitInfo.FTP_TYPE_BUS) {
             return resources.getString(R.string.transit_orca_station_coach,
                     Long.toString(getCoachNumber()));
+        } else if (getFTPType() == OrcaTransitInfo.FTP_TYPE_HANDHELD) {
+            return resources.getString(R.string.transit_orca_agency_handheld,
+                    Long.toString(getCoachNumber()));
```
Comment (codebutler) [inline farebot-transit-orca/src/main/java/com/codebutler/farebot/transit/orca/OrcaTrip.java]: remove `Long.toString(getCoachNumber()));`

**Comment by codebutler** [comment]
Thanks!

**Pull request merged**